plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -54,39 +54,39 @@ module.exports = ({containerConfig, appConfig}) => {
       </Row>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {(fields.resources.cpus != null && fields.resources.cpus > 0) && (
```

Comment:
I'm really not a fan of this merging logic and view declaration :/
Not a show stopper here, but let's have a conversation about it